exercises/1-the-manual-menace/README.md

```diff
@@ -225,23 +220,26 @@ openshift_cluster_content:
   content:
   - name: "nexus"
     namespace: "{{ ci_cd_namespace }}"
-    template: "{{ playbook_dir }}/templates/nexus.yml"
+    template: "https://raw.githubusercontent.com/rht-labs/openshift-templates/v1.3/nexus/nexus-deployment-template.yml"
```

Comment:
Is there a reason to target `v1.3` vs `v1.4.8` (latest)?

Reply:
Fixed - pointing to `v1.4.8`